frontend/src/scenes/paths/NewPaths.tsx

```diff
@@ -213,15 +257,16 @@ export function NewPaths({ dashboardItemId = null, filters = null, color = 'whit
 
         link.append('g')
             .append('path')
+            // @ts-expect-error
```

Comment:
D3 + typing is such a pain 😢. I think this is fine as long as you put a sentence here explaining why